src/parser/FStar.Parser.Dep.fs

```diff
@@ -1605,8 +1651,16 @@ let print_full (deps:deps) : unit =
               in
               if Options.should_extract (lowercase_module_name file_name)
               then
-                let cmx_files = String.concat "\\\n\t" cmx_files in
+                let cmx_files =
+                  if ninja then
+                    String.concat " " cmx_files
```

Comment:
seems like this pattern of concatenating with either `"  "` or `"\\\n\t"` could be hoisted as a helper